src/main/java/edu/harvard/iq/dataverse/dataaccess/S3AccessIO.java

```diff
@@ -1043,6 +1043,14 @@ private static AmazonS3 getClient(String driverId) {
     		// get a standard client, using the standard way of configuration the credentials, etc.
     		AmazonS3ClientBuilder s3CB = AmazonS3ClientBuilder.standard();
 
+    		ClientConfiguration cc = new ClientConfiguration();
+    		Integer poolSize = Integer.getInteger("dataverse.files." + driverId + ".connection-pool-size");
```

Comment:
this can be simplified a little:
public static Integer getInteger(String nm, int val) where val is the default (so no null check needed)